src/main/avro/behov.avdl

```diff
@@ -1,14 +1,18 @@
 @namespace("no.nav.dagpenger.events.avro")
 protocol Dagpenger {
-  enum JournalpostType {
-      NY, GJENOPPTAK, ETTERSENDING, MANUELL, UKJENT
+
+  record Søknad {
+    union { null, string } vedtakstype = null;
```

Comment:
We could use enums here (`Ny`, `Gjenopptak`). I do have a bad experience with enums - as they are hard to change when they go out in the wild. But, they could also be very helpful, if they are stable. 

Same with `rettighetsType` (`Ordinær`, `Perm`, `Fisk`.. )